javaslang/src-gen/main/java/javaslang/collection/ArrayType.java

```diff
@@ -68,7 +68,6 @@ default Object copyRange(Object array, int from, int to) {
     /** Repeatedly group an array into equal sized sub-trees */
     default Object grouped(Object array, int groupSize) {
         final int arrayLength = lengthOf(array);
-        assert arrayLength > groupSize;
```

Comment:
removed asserts from `Vector` as it's stable enough and it may hinder inlining, even if turned off :/